src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/attachment/v4_9/CloudStackAttachmentPluginTest.java

```diff
@@ -59,253 +59,140 @@
-        } finally {
-            // verify
-            PowerMockito.verifyStatic(CloudStackUrlUtil.class, VerificationModeFactory.times(1));
-            CloudStackUrlUtil.sign(Mockito.any(URIBuilder.class), Mockito.anyString());
 
-            Mockito.verify(this.client, Mockito.times(1)).doGetRequest(Mockito.anyString(),
-                    Mockito.any(CloudStackUser.class));
-        }
-    }
-    
-    // test case: When calling the requestInstance method passing some invalid argument, an
-    // FogbowException must be thrown.
-    @Test(expected = FogbowException.class)
-    public void testAttachRequestInstanceThrowFogbowException()
-            throws HttpResponseException, FogbowException {
-        
+    // test case: When calling the doRequestInstance method with secondary methods mocked,
+    // it must verify if It returns the instanceId(jobId).
+    @Test
+    public void testDoRequestInstanceSuccessfully() throws FogbowException, HttpResponseException {
         // set up
-        PowerMockito.mockStatic(CloudStackUrlUtil.class);
-        PowerMockito
-                .when(CloudStackUrlUtil.createURIBuilder(Mockito.anyString(), Mockito.anyString()))
-                .thenCallRealMethod();
+        AttachVolumeRequest request = new AttachVolumeRequest.Builder().build("");
 
-        Mockito.when(
-                this.client.doGetRequest(Mockito.anyString(), Mockito.any(CloudStackUser.class)))
-                .thenThrow(new HttpResponseException(HttpStatus.SC_BAD_REQUEST, null));
+        PowerMockito.mockStatic(CloudStackCloudUtils.class);
+        String resposeStr = "anything";
+        PowerMockito.when(CloudStackCloudUtils.doGet(
+                Mockito.eq(this.client),
+                Mockito.eq(request.getUriBuilder().toString()), Mockito.eq(cloudStackUser))).
+                thenReturn(resposeStr);
 
-        try {
-            // exercise
-            AttachmentOrder order = new AttachmentOrder(FAKE_VIRTUAL_MACHINE_ID, FAKE_VOLUME_ID, null);
+        PowerMockito.mockStatic(AttachVolumeResponse.class);
+        String jobIdExpexted = "jobIdExpected";
+        AttachVolumeResponse response = Mockito.mock(AttachVolumeResponse.class);
+        Mockito.when(response.getJobId()).thenReturn(jobIdExpexted);
+        PowerMockito.when(AttachVolumeResponse.fromJson(Mockito.eq(resposeStr))).thenReturn(response);
 
-            this.plugin.requestInstance(order, this.cloudUser);
-        } finally {
-            // verify
-            PowerMockito.verifyStatic(CloudStackUrlUtil.class, VerificationModeFactory.times(1));
-            CloudStackUrlUtil.sign(Mockito.any(URIBuilder.class), Mockito.anyString());
+        // exercise
+        String jobId = this.plugin.doRequestInstance(request, this.cloudStackUser);
 
-            Mockito.verify(this.client, Mockito.times(1)).doGetRequest(Mockito.anyString(),
-                    Mockito.any(CloudStackUser.class));
-        }
+        // verify
```

Comment:
verify if cloudStackUtils.doGet has been called. PS: this method should be renamed as commented in other pr. Just to make a note.